helloworld-jest-test-sample/README.md

```diff
@@ -0,0 +1,19 @@
+# Hello World Jest Test Sample
+
+This is the Hello World example with jest integration tests setup.
+
+You can find the accompanying documentation at:
+
+- https://code.visualstudio.com/api/working-with-extensions/testing-extension#testing-extension-with-jest
+- https://github.com/bmealhouse/vscode-jest-test-runner
```

Comment:
We don't usually link to personal repos — sorry.

Reply:
No problem, that makes sense.